qa-tests-backend/src/test/groovy/IntegrationsSplunkViolationsTest.groovy

```diff
@@ -33,7 +33,7 @@ class IntegrationsSplunkViolationsTest extends BaseSpecification {
     private static final String ASSETS_DIR = Paths.get(
             System.getProperty("user.dir"), "artifacts", "splunk-violations-test")
     private static final String PATH_TO_SPLUNK_TA_SPL = Paths.get(ASSETS_DIR,
-    "2021-07-23-TA-stackrox-1.2.0-input-validation-patched.spl")
+    "2022-10-19-TA-stackrox-1.2.2.spl")
```

Comment:
Hm. Unfortunately there are test failures exactly with Splunk. `IntegrationsSplunkViolationsTest: Verify Splunk violations: StackRox violations reach Splunk TA`

Without investigating deeper, my first hunch: could it be because our Splunk version is just too old for our TA that's built with fresh SDK?

Reply:
The test output shows:
```
    10:43:40 | INFO  | IntegrationsSplunkViolationsTest | Attempt 13 to get violations from Splunk
    10:43:40 | DEBUG | SplunkUtil                | {"sid":"1666176220.19"}
    10:43:40 | DEBUG | SplunkUtil                | New Search created: 1666176220.19
    10:43:56 | INFO  | IntegrationsSplunkViolationsTest | Attempt 14 to get violations from Splunk
    10:43:56 | DEBUG | SplunkUtil                | {"sid":"1666176236.20"}
    10:43:56 | DEBUG | SplunkUtil                | New Search created: 1666176236.20
    10:44:11 | ERROR | Helpers                   | An exception occurred in test
    org.spockframework.runtime.ConditionNotSatisfiedError: Condition not satisfied:

    !results.isEmpty()
    ||       |
    |[]      true
    false
```
Which means the test tried 14 times to get violations from Splunk (overall around 4 minutes) but they did not appear there.

Reply:
Looks like you'd need to investigate further what happens with our TA and Splunk during the test.

Reply:
As discussed in the meeting, I tried to get around spinning a whole cluster to generate the spl and therefore renamed the tgz to spl (spl are also tgz files with a different extension).
As this failed, I deployed a cluster and generated the real spl firectly from the Splunk AddOn Builder